File: C#/s/s/Location.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace s
{
    public class Location
    {
        private int row, col, i;
        public Location(int row, int col, int i)
        {
            this.row = row;
            this.col = col;
            this.i = i;
        }
        public int Row()
        {
            return row;
        }
        public int Col()
        {
            return col;
        }
        public int Index()
        {
            return i;
        }

        public override string ToString()
        {
            return "位置" + (row+1) + "行" + (col+1) + "列，第" + (i+1) + "个字符串";
        }
    }
}

File: JAVA/s/src/main/java/s/Location.java
package s;

public class Location{
    public Location(int r,int c,int index){
        this._r=r;
        this._c=c;
        this._i=index;
    }
    private int _r,_c;
    public int row(){
        return _r;
    }
    public int column(){
        return _c;
    }
    private int _i;
    public int index() {
    	return _i;
    }
    
    private int _l;
    public void setLength(int l) {
    	_l=l;
    }
    public int length() {
    	return _l;
    }
    @Override
    public String toString(){
        return "{第"+(_r+1)+"行，第"+(_c+1)+"列}";
    }
}